spec/acceptance/pe_status_check_spec.rb

```diff
@@ -284,6 +285,27 @@ class {'pe_status_check':
         expect(result.stdout).to match(%r{false})
         run_shell('touch -d "1 day ago"  /opt/puppetlabs/server/pe_build')
       end
+      context 'S0035' do
```

Comment:
true is already checked earlier in the module for all tests, we only need to explicitly check false